docs/topics/opt-in-requirements.md

```diff
@@ -219,11 +269,82 @@ For Maven, it would be:
 
 To opt in to multiple APIs on the module level, add one of the described arguments for each opt-in requirement marker used in your module.
 
-## Require opt-in for API 
```

Comment:
could we have a short description similar to the one in Opt in to API﻿ section?

Reply:
Do you mean something like:

```
If a library author marks a declaration from their library's API as requiring opt-in to extend, you must give explicit consent before you can extend it in your code.
```

Reply:
exactly! 👍